app/src/main/java/com/amaze/filemanager/ui/activities/superclasses/ThemedActivity.java

```diff
@@ -105,23 +105,22 @@ public void initStatusBarResources(View parentView) {
       } else {
         window.addFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
       }
+      if (getBoolean(PREFERENCE_COLORED_NAVIGATION)) {
```

Comment:
@EmmanuelMess I hope transfer the code to the right block as requested hope its fine now